### PR TITLE
Update line 29, according to https://office.visualstudio.com/OC/_work…

### PR DESCRIPTION
**Pull request by fishgwo**
Update line 29, according to https://office.visualstudio.com/OC/_workitems/edit/4554929

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/organization-assets-library.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ If your organization needs to store and manage files for all your users to use,
 
     ![Selecting an image to add to a SharePoint page](media/image-library.png)
 
-- **Office templates**. When a user selects to create a new Office file (from Office.com or the Word, Excel, or PowerPoint desktop app), the user can select the tab for your organization to see the templates. (To use this feature on Office.com, users need to be assigned a license to Office 365 E3 or E5. The templates aren't available from the New menu. To create a doc from a template, go to Office.com > app start page > Office Template Library.)
+- **Office templates**. When a user selects to create a new Office file (from Office.com or the Word, Excel, or PowerPoint desktop app), the user can select the tab for your organization to see the templates. (Currently this feature is only available for PowerPoint on Office.com, users need to be assigned a license to Office 365 E3 or E5. The templates aren't available from the New menu. To create a doc from a template, go to Office.com > app start page > Office Template Library.)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/organization-assets-library.md:29]:
```suggestion
- **Office templates**. When a user selects to create a new Office file (from Office.com or the Word, Excel, or PowerPoint desktop app), the user can select the tab for your organization to see the templates. (Currently, this feature is only available for PowerPoint on Office.com and users need to be assigned a license to Office 365 E3 or E5. The templates aren't available from the New menu. To create a doc from a template, go to Office.com > app start page > Office Template Library.)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by fishgwo** [comment]
@kaarins  Is there progress on this, can we release document update?

**Comment by kaarins** [comment]
Thank you! I made additional updates after discussing with PM and product marketing.

**Pull request closed without merging**